lib/util/action.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 module.exports = (key, isSelect) => {
+  if (key.meta) return;
```

Comment:
The function returns `false` at the end. Shouldn't it also return `false` here? Or won't it have the needed effect?
If it should be as is, then I would approve it 😃

Reply:
In fact, when nothing (or `undefined`) is returned the bell function is called and when `false` is returned it calls `this._(str, key)`.

Though, if you think it's better to return false instead of undefined I can easily change it.

https://github.com/terkelg/prompts/blob/0188d6a3d7f5ba95e94831de1f26cdfca7cb8ec3/lib/elements/prompt.js#L27-L36